addons/mass_mailing/models/mail_mail.py

```diff
@@ -62,11 +62,11 @@ def _prepare_outgoing_body(self):
             )
         return body
 
-    def _prepare_outgoing_list(self, recipients_follower_status=None):
+    def _prepare_outgoing_list(self, mail_server=False, recipients_follower_status=None):
         """ Update mailing specific links to replace generic unsubscribe and
         view links by email-specific links. Also add headers to allow
         unsubscribe from email managers. """
-        email_list = super()._prepare_outgoing_list(recipients_follower_status)
+        email_list = super()._prepare_outgoing_list(mail_server, recipients_follower_status)
```

Comment:
I prefer to keep named parameters when calling super, like
```suggestion
        email_list = super()._prepare_outgoing_list(mail_server=mail_server, recipients_follower_status=recipients_follower_status)
```